mockito-core/src/main/java/org/mockito/plugins/MockMaker.java

```diff
@@ -171,9 +171,8 @@ default <T> StaticMockControl<T> createStaticMock(
                                 + getClass().getSimpleName()
                                 + " does not support the creation of static mocks",
                         "",
-                        "Mockito's inline mock maker supports static mocks based on the Instrumentation API.",
-                        "You can simply enable this mock mode, by placing the 'mockito-inline' artifact where you are currently using 'mockito-core'.",
-                        "Note that Mockito's inline mock maker is not supported on Android."));
+                        "Ensure your MockMaker implementation supports this feature.",
+                        "Note that static mocks maker is not supported on Android."));
```

Comment:
These messages LGTM. We have some test assertions that verify the message contents. Please update these accordingly.

Reply:
I updated the messages on the test.
Thanks for the review, @TimvdLippe 🙇